setup-r-dependencies/action.yaml

```diff
@@ -116,24 +119,63 @@ runs:
         shell: Rscript {0}
         working-directory: ${{ inputs.working-directory }}
 
-      - name: Restore R package cache
-        if: inputs.cache == 'true'
+      - name: R package cache
+        if: inputs.cache != 'false'
         uses: actions/cache@v4
         with:
           path: |
             ${{ env.R_LIBS_USER }}/*
             !${{ env.R_LIBS_USER }}/pak
           key: ${{ format('{0}-{1}-{2}-{3}', steps.install.outputs.os-version, steps.install.outputs.r-version, inputs.cache-version, hashFiles(format('{0}/.github/pkg.lock', inputs.working-directory ))) }}
-          restore-keys: ${{ steps.install.outputs.os-version }}-${{ steps.install.outputs.r-version }}-${{inputs.cache-version }}-
+          restore-keys: ${{ format('{0}-{1}-{2}-', steps.install.outputs.os-version, steps.install.outputs.r-version, inputs.cache-version) }}
+          # Used to save package installation on troublesome Actions
+          save-always: ${{ inputs.cache == 'always' }}
```

Comment:
@gaborcsardi Should we make `cache="always"` the default value? Because it is independent of the tests, it feels like we should help as much as possible.

Reply:
Not yet. Later maybe, when we'll have the pruning.